galsim/chromatic.py

```diff
@@ -140,9 +355,14 @@ def drawImage(self, bandpass, image=None, integrator=None, **kwargs):
 
         The task of drawImage() in a chromatic context is to integrate a chromatic surface
         brightness profile multiplied by the throughput of `bandpass`, over the wavelength interval
-        indicated by `bandpass`.
-
-        Several integrators are available in galsim.integ to do this integration.  By default,
+        indicated by `bandpass`.  This integration will take place either via brute-force drawing
+        the image at each wavelength, or via an optimized version of the routine that uses some
```

Comment:
"each wavelength" is a bit of a non sequitur at this point in the doc.  You haven't talked about any list of wavelengths yet.  Also, could probably pare this down a bit and just reference `setupInterpolation` for more details.